### ISSUE TITLE
Incorrect Hessian in call_highs_from_c.c

### ISSUE THREAD

**Issue by spakin** [ERGO-Code/HiGHS#1070]
I'm new to QP, but it sure looks to me that the Hessian specified in [`examples/call_highs_from_c.c`, lines 211–215](https://github.com/ERGO-Code/HiGHS/blob/4ebb25d4b1e21addaf48fb0e4c8dd0bca04efff6/examples/call_highs_from_c.c#L211-L215) is specifying the Hessian as a square matrix rather than the triangular matrix it claims to be.  Given $f = \frac{1}{2} (2 x_1^2 - 2 x_1 x_3 + 0.2 x_2^2 + 2 x_3^2)$ from the problem statement, I compute a Hessian of
```math
\begin{bmatrix}
  2 & 0 & -1 \\
  0 & 0.2 & 0 \\
  -1 & 0 & 2
\end{bmatrix}
\; ,
```
which is exactly what is represented by the current code's
```C
const int q_format = kHighsHessianFormatTriangular;
const int q_num_nz = 5;
const int q_start[3] = {0, 2, 3};
const int q_index[5] = {0, 2, 1, 0, 2};
const double q_value[5] = {2.0, -1.0, 0.2, -1.0, 2.0};
```
as a _square_ matrix.  As a _triangular_ matrix, I'd think this should be
```C
const int q_format = kHighsHessianFormatTriangular;
const int q_num_nz = 4;
const int q_start[3] = {0, 2, 3};
const int q_index[4] = {0, 2, 1, 2};
const double q_value[4] = {2.0, -1.0, 0.2, 2.0};
```

Am I correct, or am I misunderstanding how to formulate QP problems in HiGHS?

**Comment by jajhall** [comment]
You're correct, and thanks for pointing this out. We ought to run these example programs sometime!

**Comment by spakin** [comment]
Thanks for confirming.  Would you like me to submit a pull request to make that change?

**Comment by jajhall** [comment]
No thanks, I'll handle it

**Comment by spakin** [comment]
OK, cool.

**Comment by jajhall** [comment]
The correction will work its way to HiGHS v1.5.0.

In case you're interested, internally HiGHS ignored the upper-triangular entry of -1 in row 0 of column 2 and works with the 4-nonzero triangular form you suggested.

**Comment by spakin** [comment]
Thanks for the explanation and the correction.
